src/Error/ExceptionRenderer.php

```diff
@@ -184,7 +184,6 @@ public function render()
                 'format' => 'array',
                 'args' => false
             ]);
-            $viewVars['_serialize'][] = 'trace';
```

Comment:
Ah, it wasn't my change, that's why I was confused.
Well, not having them at all (and logging it instead?) is also OK for me, but it wouldn't have to be this way with (my) previous code. In many cases it works fine, but in those it doesn't my solution is a more graceful fallback than not dislaying them at all - maybe.